### DISCUSSION TITLE
Add export options to CSV

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#24936] [Feature Requests]
### Describe the Improvement

When exporting CSVs, it's currently not possible to certain options like delimiter or "wrap in quotes". This hinders the usability, as certain copies of spreadsheet software require some of these options to be set in specific ways.

**Comment by paescuj** [comment]
Related issue #16818, discussion #12456 and earlier attempt #16835

**Comment by w0kyj** [comment]
Export options from tableplus in case it helps:
<img width="342" alt="Screenshot 2024-02-08 at 3 26 36 PM" src="https://github.com/directus/directus/assets/40695629/a515b020-6ac0-4c16-8d9b-fe943fb235b9">

**Comment by LevelUpGit** [comment]
Hi, it would be also good thing to add default option, and possibly option to hide certain export methods, just to simplify experience.

**Comment by vwasteels** [comment]
Microsoft Excel on Mac is expecting semicolon

**Comment by vwasteels** [comment]
it would work if the separator were declared at the first line like this :

```
sep=,
COLUMN1,COLUMN2
data1,data2
data1,data2
```

that would make the CSV export openable in Excel easily

**Comment by rijkvanzanten** [comment]
> Microsoft Excel on Mac is expecting semicolon

This depends on locale settings doesn't it? I remember Numbers.app behaving the same when my computer was set to Dutch rather than English (US)

**Comment by LevelUpGit** [comment]
Main issue is when user default delimiter is ; on computer, anything downloaded will not open right for them. If they are less tech oriented, and don't know how to import csv into excel we have an issue. For example all our users use ; as delimiter, and it would be great if it would be possible to change it application wide.
